README.md

```diff
@@ -59,16 +76,22 @@ To see the Java Agent in action with sample applications, see our
 ### Basic Configuration
 
 The service name resource attribute is the only configuration option
-that typically needs to be specified. You can set it by adding a `service.name`
+that is required by the javaagent. You can set it by adding a `service.name`
```

Comment:
I'm not sure about the wording - it sounds as if the agent don't run without the property (which is not true). I'd change to "should be specified".

Reply:
Eh, true -- it should be required though 😛